### ISSUE TITLE
Auto select correct spelling when typing.

### ISSUE THREAD

**Issue by weston89** [florisboard/florisboard#414]
Currently when using the dictionary predictions, if you spell a word wrong, it will keep the incorrect spelling. On other keyboards I use, it will fix simple spelling and capitilization errors as I type. Is this somethin that will be implemented eventually?

Also, it doesn't currently recognize that "I" should be capitalized.

**Comment by patrickgold** [comment]
Yes, definitely. Just don't know exactly when, but phase 3 seems most probable for me.

**Comment by mizzunet** [comment]
+ on this

**Comment by blankstaticscreen** [comment]
i realize it's been a while since this was proposed in the first place, but the guy mentions

> it doesn't currently recognize that "I" should be capitalized.

and i never capitalize when typing unless it's like, an actual email or something. it's just a personal preference. if i have auto-capitalize off, i'd really like it to NOT capitalize i, or any proper noun, please!!
